workshop/SCHEDULE.md

```diff
@@ -5,15 +5,85 @@ nav_title: Schedule
+This schedule is subject to minor changes and will be updated with additional materials before the workshop.
 
+The workshop will takes place from August 19-22, 2024 at the University of Minnesota.
+We will meet each day at 717 Delaware Street SE,
+Minneapolis, MN 55414 in Room 105.
 
-| Time        | Topic                                          |
+*Note: All times are CDT.*
+
+| Time        | Topic                                          | 
+|-------------|------------------------------------------------|
+| **Day 1**   | **2024-08-19 <br> Intro to R and the Tidyverse <br> Intro to Reproducible Research Practices**|
+| 8:30 AM     | Arrive and check-in <br> Breakfast and coffee available | 
+| 9:00 AM    | Welcome, introductions and getting started |
+| 10:00 AM     | [Introduction to R and RStudio](https://htmlpreview.github.io/?https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/intro-to-R-tidyverse/01-intro_to_base_R.nb.html) | 
+| 11:15 AM   | Coffee break |
+| 11:30 AM     | [Introduction to `ggplot2`](https://htmlpreview.github.io/?https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/intro-to-R-tidyverse/02-intro_to_ggplot2.nb.html) | 
+| 12:30 PM     | Lunch break | 
+| 1:30 PM     | [Introduction to the `tidyverse`](https://htmlpreview.github.io/?https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/intro-to-R-tidyverse/03-intro_to_tidyverse.nb.html) |
+| 2:30 PM     | Coffee break |
+| 2:45 PM     | Questions and consultation time | 
+|             | [Exercise: Introduction to base R](https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/intro-to-R-tidyverse/exercise_01-intro_to_base_R.Rmd)  | 
+|             | [Exercise: Introduction to R](https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/intro-to-R-tidyverse/exercise_02-intro_to_R.Rmd)  | 
+|             | [Exercise: Introduction to the `tidyverse`, Part 1](https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/intro-to-R-tidyverse/exercise_03a-intro_to_tidyverse.Rmd)  | 
+|             | [Exercise: Introduction to the `tidyverse`, Part 2](https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/intro-to-R-tidyverse/exercise_03b-intro_to_tidyverse.Rmd)  | 
+| 3:30  PM    | Introduction to reproducible research practices |
+| 4:00  PM    | Project organization  |
+| 5:00  PM    | Adjourn |
+
+| Time        | Topic                                          | 
+|-------------|------------------------------------------------|
+| **Day 2**   | **2024-08-20 <br> Reproducible Research Practices <br> Intro to Bulk RNA-Seq**  |
+| 8:30 AM     | Breakfast and coffee available |
+| 9:00 AM     | Questions and review |
+| 9:30 AM     | Introduction to UNIX and the command line |
+| 10:30 AM    | Coffee break |
+| 10:45 AM    | Introduction to Git / Forking and cloning |
```

Comment:
So actually one question here is if we want to do the data sharing slides too. On one hand I think they are good to present since a lot of folks may not have thought much about data sharing previously, but on the other hand the schedule is already very full! @jaclyn-taroni what do you think?

Reply:
What if we don't pick between those two options, and instead, we leave it in as something we'll potentially skip if we are running behind?

Reply:
Update: I was viewing this through the GitHub app so I didn't have all the context. I don't feel super strongly that we need to insert it if we think it would be difficult.

Reply:
I think it would probably be difficult, so let's not insert it. But we can keep it in our back pocket if, by some miracle, we end up well ahead of schedule.